ocsp/responder.go

```diff
@@ -19,6 +20,7 @@ import (
 
 	"github.com/cloudflare/cfssl/log"
 	"github.com/jmhodges/clock"
+	"github.com/cloudflare/cfssl/certdb"
```

Comment:
According to `gofmt`, you need to move this line two lines above, so the alphabetic order of import block is maintained.